app/controllers/concerns/sorting_concern.rb

```diff
@@ -0,0 +1,30 @@
+module SortingConcern
+  extend ActiveSupport::Concern
+
+  protected
+
+  # Sorting params example: capture_time:desc,title:asc
+  def sortation(sort_params)
+    return if sort_params.nil?
+
+    sort_params.split(',')
+               .map { |sort| ordering(sort) }
+               .join(', ')
+    end
+  end
+
+  private
+
+  def ordering(sort)
+    key, direction = sort.split(':')
+
+    "#{key} #{sanitize_direction(direction)}"
```

Comment:
This does not seem safe. [ActiveRecord does not sanitize single string arguments to `order()`](https://github.com/rails/rails/blob/v5.1.4/activerecord/lib/active_record/sanitization.rb#L64-L70). Maybe better to produce a hash of `key => direction` here so we get the benefit of AR’s built-in sanitization and safety? Or quote the key?